src/v/archival/ntp_archiver_service.h

```diff
@@ -87,9 +87,6 @@ class ntp_archiver {
     /// Get revision id
     model::initial_revision_id get_revision_id() const;
 
-    /// Get timestamp
-    const ss::lowres_clock::time_point get_last_upload_time() const;
-
```

Comment:
Just because it was dead code.

I'll add back in getters for segment+manifest last upload time, for the benefit of #9237.  Up to you if you want to max() the two, or represent them separately.